common/stepactions/analyze-test-results/0.1/README.md

```diff
@@ -0,0 +1,14 @@
+# analyze-test-results stepaction
+
+This StepAction runs "analyze-test-results" command from qe-tools binary against supplied OCI artifact that contains logs and test results.
```

Comment:
I'm kind of wondering if we should include a link to the cmd in the qe-tools repo? And maybe append  the same sentence "which tries to identify the cause of the failure and provide the result of the analysis"

Reply:
thanks, updated